velero-plugins/common/types.go

```diff
@@ -25,7 +25,8 @@ const (
 	MigrationApplicationLabelKey   string = "app.kubernetes.io/part-of"
 	MigrationApplicationLabelValue string = "openshift-migration"
 )
-
+// Skip buildconfig configmap restore
+const SkipBuildConfigConfigMapRestore string = "openshift.io/skip-buildconfig-configmap-restore"
```

Comment:
We've been using "oadp.openshift.io" for recently-added annotations. "openshift.io" was what we started with, so we've left the old ones as they were, but I'd add the "oadp" prefix here.